spec/raindrop_spec.rb

```diff
@@ -0,0 +1,22 @@
+require './raindrop.rb'
+
+RSpec.describe "raindrop" do
+  it 'should return pling when input is divisible by 3' do
+    expect(raindrop(6)).to eq('pling')
+  end
+  it 'should return plang when input is divisible by 5' do
+    expect(raindrop(10)).to eq('plang')
+  end
+  it 'should return plong when input is divisible by 7' do
+    expect(raindrop(14)).to eq('plong')
+  end
+  it 'returns "plingplang" for when input  both 3 and 5' do
```

Comment:
some chore may try using `rubocop`
```suggestion
  it 'returns "plingplang" for when input both 3 and 5' do
```
ref: https://github.com/rubocop/rubocop